lib/multi_session.rb

```diff
@@ -9,6 +9,9 @@ module MultiSession
   mattr_accessor :expires
   @@expires = nil
 
+  mattr_accessor :credentials_strategy
+  @@credentials_strategy = :credentials
```

Comment:
Can you default this to `:credentials` or `:secrets` depending on `Rails.application.respond_to? :credentials`?

Reply:
Also could you add this option to the install generator template (commented out).